tests/integration/telemetry/tracing/clienttracing/client_tracing_test.go

```diff
@@ -84,4 +84,17 @@ func setupConfig(cfg *istio.Config) {
 	cfg.Values["tracing.provider"] = "zipkin"
 	cfg.Values["global.enableTracing"] = "true"
 	cfg.Values["global.disablePolicyChecks"] = "true"
+
+	// TODO not needed once https://github.com/istio/istio/issues/20137 is in
+	cfg.ControlPlaneValues = `
```

Comment:
@douglas-reid mind taking a look? This is to workaround a bug/feature introduced by a recent change in operator, not intended to change the test

Reply:
Looks reasonable to me. Should we add something to filed bug about confirming that these values have been removed and the tests pass as expected?

Reply:
good point. It would be cool if github linked issues to code comments.. I added a comment on the issue